projects/aca-content/src/lib/components/search/search-save/dialog/save-search-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<h2 class="aca-save-search-dialog__header" mat-dialog-title>{{"APP.BROWSE.SEARCH.SAVE_SEARCH.MODAL_HEADER" | translate}}</h2>
```

Comment:
Do you need that class? It’s unused. If we need this for testing then data-automation-id should be used instead.